code/utils/scripts/lin_reg_plots.py

```diff
@@ -23,19 +24,18 @@
 smooth_data           = final_data + 'smooth/'
 hrf_data              = final_data + 'hrf/'
 
-
-#sys.path.append(os.path.join(os.path.dirname(__file__), "../functions/"))
```

Comment:
You don't have to. Something along the lines of "sys.path.append(location_of_functions)" where location_of_functions= ../functions or whatever has been working for me. I don't remember why I couldn't get it to cooperate earlier this semester, which is why a lot of older scripts have the os.path.join thing. "sys.path.append(location_of_functions)" is cleaner and easier, so I was going to update all the scripts with that later myself if you guys hadn't gotten to it already.